scripts/install.py

```diff
@@ -137,22 +144,29 @@ def main():
         install_gc3pie_from_pypi(options.target, options.features)
 
     print("""
-===============================
-Installation of GC3Pie is done!
-===============================
-
-In order to work with GC3Pie you have to enable the virtual
-environment with the command:
-
-    . {target}/bin/activate
-
-You need to run the above command on every new shell you open before
-using GC3Pie commands, but just once per session.
+            ===============================
+            Installation of GC3Pie is done!
+            ===============================
+
+            In order to work with GC3Pie you have to enable the virtual
+            environment with the command:
+    """)
+    if not default.ANACONDA:
```

Comment:
I find this (lines 154--160) a bit hard to read; why not make a new template parameter?

For instance::

        if default.ANACONDA:
          activation_cmd = 'source activate {target}'
        else:
          activation_cmd = '. {target}/bin/activate'
        activation_cmd = activation_cmd.format(target=target)
        print('''
            ...
            In order to work with GC3Pie you have to enable the virtual
            environment with the command:
                {activation_cmd}
        '''.format(activation_cmd=activation_cmd))